Zend/tests/exception_types/010.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
```

Comment:
@nikic @bwoebi Optimizer gives me a headache, I'm super unfamiliar with it (on purpose) ... urm ... help ?